.github/workflows/release.yml

```diff
@@ -40,6 +56,68 @@ jobs:
       run: |
         python setup.py bdist_wheel sdist
 
+    - name: Save package
+      if: matrix.python == '3.7'
+      uses: actions/upload-artifact@v1
+      with:
+        name: models-package
+        path: dist
+
+  test:
+    name: Test
+    needs: [build]  # needs the package artifact created from 'build_package' job.
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.6', '3.7']
+
+    steps:
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python }}
+
+    - name: Install requirements
+      run: |
+        pip install --upgrade pip setuptools wheel
+
+    - name: Download models package
+      uses: actions/download-artifact@v1
+      with:
+        name: models-package
+        path: dist
+
+    - name: Install models package
+      run: |
+        pip install $(ls dist/*.whl)
+
+    - name: Ensure models plugin found
+      run: |
+        python -c 'from allennlp.common.plugins import discover_plugins; assert "allennlp_plugins.allennlp_models" in list(discover_plugins())'
```

Comment:
👍

Reply:
I love the thoroughness of this. Something went wrong, so here is some code to make sure next time we get an error first. Awesome!